src/tests/topp/ProteomicsLFQ_5_out.csv

```diff
@@ -25,6 +25,7 @@ P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,2,NA,0,L,1,1,1,1,
 P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,3,NA,0,L,1,1,1,1,1.240657e07,"BSA1_F1.mzML"
 P02769|ALBU_BOVIN,EC(Carbamidomethyl)C(Carbamidomethyl)DKPLLEK,3,NA,0,L,3,3,5,1,2.598459e06,"BSA3_F1.mzML"
 P02769|ALBU_BOVIN,ETYGDMADC(Carbamidomethyl)C(Carbamidomethyl)EK,2,NA,0,L,2,2,3,1,6.884387e05,"BSA2_F1.mzML"
+P02769|ALBU_BOVIN_rev,EVEAIC(Carbamidomethyl)HSK,3,NA,0,L,1,1,1,1,1.358151e06,"BSA1_F1.mzML"
```

Comment:
Should we skip export to MSstats?

Reply:
hmm probably, for now. Otherwise, people need to filter in the R scripts